app/shared/upload-picture/upload-picture.component.ts

```diff
@@ -106,29 +121,33 @@ export class UploadPictureComponent implements OnInit {
     }
     canvas.width = width;
     canvas.height = height;
-    var ctx = canvas.getContext("2d");
-    var dataUrl = canvas.toDataURL('image/jpeg');
+    let dataUrl = canvas.toDataURL('image/jpeg');
     return dataUrl
   }
 
   removePicture(): void {
     this.uploadedImage = '';
     this.previewedImage = '';
+    this.fileInput = '';
+    (<HTMLElement>document.getElementById('uploadedFile')).value = '';
```

Comment:
can't this be done like above? `let img = this.previewImageFile` and `img.value = ...`